activity_browser/ui/wizards/uncertainty.py

```diff
@@ -10,13 +10,17 @@
 from ...bwutils import PedigreeMatrix, get_uncertainty_interface
 from ...bwutils.uncertainty import EMPTY_UNCERTAINTY
 from ...signals import signals
+from ...logger import log
 
 
 class UncertaintyWizard(QtWidgets.QWizard):
     """Using this wizard, guide the user through selecting an 'uncertainty'
     distribution (and related values) for their activity/process exchanges.
 
     Note that this can also be used for setting uncertainties on parameters
+
```

Comment:
Is this supposed to be in this PR?

Reply:
Sorry, this slipped through the different merges that I'd made previously